### PR TITLE
Makes resurrecting and killing intellicarded AIs less painful

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR:
1. Multiplies the speed at which AI integrity restorers heal intellicarded AIs by 5.
2. Makes AI integrity restorer consoles automatically turn on an intellicard's subspace tranceiver when they heal a conscious AI inside of said intellicard.
3. Multiplies the speed at which wiping an intellicard deals damage to an AI within that intellicard by 5.

## Why It's Good For The Game

1. It currently takes 200 ticks (400 seconds, or 6.67 minutes) to return a dead AI in an intellicard to full health using an AI integrity restorer (either with the modular computer program or the dedicated AI restoration console). This PR will bring that time down to a much more reasonable 40 ticks (80 seconds, or 1.33 minutes), since so few people currently even bother healing dead AIs and because nobody wants to wait 6+ minutes by an AI restoration console to wait until an AI is fully healed (or 3+ minutes if you want to wait until they'll be conscious). The healing process for dead AIs is PAINFULLY slow right now, and this PR should help make it a little more bearable.
2. This helps prevent "forgotten shade in a soulstone"-like scenarios; if you forget about an AI that you put inside of an AI integrity restorer (or die before you can remove it from the console), it can beg for help from the entire station instead of just from whomever happens to pass by the RD's office (and god help you if someone closed the shutters...).
3. It currently takes 200 ticks (400 seconds, or 6.67 minutes) to wipe an AI using an intellicard's AI-wiping feature. This PR will bring that time down to a much more reasonable 40 ticks (80 seconds, or 1.33 minutes), which means that it (usually) won't be faster to _build a new AI core_, transfer the AI inside of the intellicard into that AI core, and then bludgeon that AI core with a blunt object until the AI inside of it dies than it is to wait out the incredibly long wiping process.

## Changelog
:cl: ATHATH
balance: Multiplied the speed at which AI integrity restorers heal intellicarded AIs by 5.
balance: Made AI integrity restorer consoles automatically turn on an intellicard's subspace tranceiver when they heal a conscious AI inside of said intellicard.
balance: Multiplied the speed at which wiping an intellicard deals damage to an AI within that intellicard by 5.
/:cl:

### CODE REVIEW

**Comment by carshalash** [comment]
Thank you jesus

**Comment by ATH1909** [comment]
oh god why is this causing a merge conflict how do I fix this help

File: code/modules/modular_computers/file_system/programs/airestorer.dm [hunk 1]
```diff
@@ -74,13 +74,16 @@
...
+	A.adjustOxyLoss(-5, 0)
+	A.adjustFireLoss(-5, 0)
+	A.adjustToxLoss(-5, 0)
+	A.adjustBruteLoss(-5, 0)
 	A.updatehealth()
 	if(A.health >= 0 && A.stat == DEAD)
 		A.revive()
+		if(!A.radio_enabled)
+			A.radio_enabled = TRUE
+			to_chat(A, "<span class='warning'>Your Subspace Transceiver has been enabled!</span>")
```
Comment (zxaber) [inline code/modules/modular_computers/file_system/programs/airestorer.dm]: Unlike the stand-alone computer, the modular computer uses an AI card slot. Thus, the AI never actually leaves the card, and you probably don't want to update their radio_enabled value.

**Comment by Cyprex** [comment]
I like how wiping takes a while, you can just space the card while it's wiping if you really want to get rid of it or use other methods to delete them(deep-frying, cremator, possibly bashing it?). IT also lets you taunt the wiping person and such things. I'd keep it as-is

**Comment by Kingtrin** [comment]
> I like how wiping takes a while, you can just space the card while it's wiping if you really want to get rid of it or use other methods to delete them(deep-frying, cremator, possibly bashing it?). IT also lets you taunt the wiping person and such things. I'd keep it as-is, if you really want to kill an AI efficiently you can just bash in their core and card them afterwards

The point is that its a bit silly for it to be considerably faster to not use the feature than it is to use it. It's too stupid that the core, made of plasteel and reinforced glass, is quicker to destroy than a chip made of glass and steel. It's hard to see why the wiping time should remain so long when theres little cause for it.

**Comment by CPTANT** [comment]
100% yes.

I timed it, with server lag added it took me 8(!) minutes to get restored as an AI.

**Comment by terranaut1** [comment]
shut up and take my money

**Comment by Vanadiom** [comment]
Nice stuff, really would help AI players, but can you make turning off subspace tranceiver an optional tickbox, in case an antag wants to restore AI, for the "working AI" quests.
 You know, so it doesn't shout at the crew the very second It gains sentience again.

**Comment by ATH1909** [comment]
> Nice stuff, really would help AI players, but can you make turning off subspace tranceiver an optional tickbox, in case an antag wants to restore AI, for the "working AI" quests.
> You know, so it doesn't shout at the crew the very second It gains sentience again.

Mm, I could try, but I'd probably have to learn the basics of /tg/ GUIs to do that. I'll look into it after Halloween, since I have a test tomorrow and the day after tomorrow.

**Comment by actioninja** [comment]
AI restoration takes an incredibly long time considering a lot of the time the limiting factor isn't the ability to restore it.

**Pull request merged**